virtualcluster/pkg/webhook/virtualcluster/virtualcluster_webhook.go

```diff
@@ -143,6 +144,11 @@ func createValidatingWebhookConfiguration(client client.Client) error {
 	svcPort := int32(constants.VirtualClusterWebhookPort)
 	// reject request if the webhook doesn't work
 	failPolicy := admv1beta1.Fail
+	// use the serviceaccount ca file as the authority
```

Comment:
This code was removed by #55.  Did you add them back on purpose?

Reply:
Oops, thanks for catching that. I don't know how that made it back in.